.eslintrc.json

```diff
@@ -28,7 +28,6 @@
       "no-extra-semi": "warn",
       "no-case-declarations": "warn",
       "no-useless-escape" : "warn",
-      "no-prototype-builtins": "warn",
-      "no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }]
```

Comment:
Please rebase to `master` branch. :)

Reply:
This branch include files that we haven't modified. So, Should we make PR for merge?

Reply:
No, this base commit of this PR is not latest main. So you need to update your remote master branch. And rebase this branch to remote master branch. :)

Reply:
We rebased our branch to the latest main branch. And we squahes all commits into one commit, because we thought we had to merge our branch to our remote main branch. But, it was not. So we stopped since we've squashed our commits. :) 
@jyoungyun